pkg/local_object_storage/blobstor/peapod/peapod.go

```diff
@@ -0,0 +1,200 @@
+package peapod
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io/fs"
+	"path/filepath"
+	"time"
+
+	"github.com/nspcc-dev/neofs-node/pkg/local_object_storage/blobstor/common"
+	"github.com/nspcc-dev/neofs-node/pkg/local_object_storage/blobstor/compression"
+	"github.com/nspcc-dev/neofs-node/pkg/local_object_storage/peapod"
+	"github.com/nspcc-dev/neofs-node/pkg/local_object_storage/util/logicerr"
+	"github.com/nspcc-dev/neofs-node/pkg/util"
+	apistatus "github.com/nspcc-dev/neofs-sdk-go/client/status"
+	objectSDK "github.com/nspcc-dev/neofs-sdk-go/object"
+	oid "github.com/nspcc-dev/neofs-sdk-go/object/id"
+	"go.etcd.io/bbolt"
+)
+
+type storage struct {
+	path string
+	perm fs.FileMode
+
+	ppd *peapod.Peapod
+
+	compress *compression.Config
+}
+
+func New(path string, perm fs.FileMode) common.Storage {
+	return &storage{
+		path: path,
+		perm: perm,
+	}
+}
+
+func (x *storage) Open(readOnly bool) error {
+	if x.ppd != nil {
+		err := x.ppd.Close()
+		if err != nil {
+			return fmt.Errorf("close Peapod: %w", err)
+		}
+	}
+
+	err := util.MkdirAllX(filepath.Dir(x.path), x.perm)
```

Comment:
to place db into the configured path obviously. It's done in, for example, in FSTree as well https://github.com/nspcc-dev/neofs-node/blob/21e9d2acf2f23fd9132b4d3994000d4efca16c9a/pkg/local_object_storage/blobstor/fstree/control.go#L15
so that's a separate issue. I dont see any problem with mkdir, it should be done anyway, right now - easier for admin

Reply:
Well, they're both wrong. The only reason they do this is #2497, no one wants to manage things in `/srv/neofs/data0` (so _that_ is what should be configured, not `/srv/neofs/data0/bbcz`). It's kinda OK for this PR, but not OK long-term.

Reply:
agree with u, lets resolve within that issue then (as well as flock timeouts)